### DISCUSSION TITLE
HowTo Start Webserver listening on /wifi or /config to get in ConfigMode

### DISCUSSION THREAD

**Discussion by DerLang061975** [khoih-prog/ESPAsync_WiFiManager#111] [Q&A]
Hi,

I'm searching a lot but did not find anything suiteable. Despite the way of hardware button, I'd like to get into Config Mode to set Wifi credentials and timezone during runtime.

I.e. I have to change my Wifi password due to intrusion. I've now to go to all my DIY sensor, open water proof enclosures and push a button?
That's not convenient so I'd like to get into Config WebPage with a special url of my device to set a new AP and PWD.

How can I achieve this?

Many thanks for your efforts on this so far very nice code.

Regards
Joerg

**Comment by khoih-prog** [comment]
Hi @DerLang061975 

Thanks for your nice words about the library, which is better and better everyday thanks to inquisitive and creative users like you.

> I.e. I have to change my Wifi password due to intrusion. I've now to go to all my DIY sensor, open water proof enclosures and push a button?
> That's not convenient so I'd like to get into Config WebPage with a special url of my device to set a new AP and PWD.
> 

As you know, the **real-life use-cases** are much more complex than those **simple examples**, which are designed only to demonstrate the **basic usage** of the library's functions.

To enter the Config Portal forcefully, there are many ways you can achieve, besides the SW pressed, such as

1. Periodically check a website for any change or message, targeting this system, by checking UserID, etc.
2. Periodically receive messages via MQTT,  etc.
3. **Millions** of ways you can think, actively checking or passively receiving commands.

We'd appreciate if you can come up with a method to deal with this, then post here so that many other users can follow.

Best,
